tidb-lightning/tidb-lightning-data-source.md

```diff
@@ -286,6 +287,17 @@ Note that this configuration only shows how to match the parquet files exported
 
 For more information on `mydumper.files`, refer to [Match customized file](#match-customized-files).
 
+## Compressed files
+
+TiDB Lightning currently supports compressed files exported by Dumpling or compressed files that follow the naming rules. Currently, TiDB Lightning supports the following compression algorithms: `gzip`, `snappy`, and `zstd`. When the file name follows the naming rules, TiDB Lightning automatically identifies the compression algorithm and imports the file after streaming decompression, without additional configuration.
+
+> **Note:**
+>
+> - Because TiDB Lightning cannot concurrently decompress a single large compressed file, the size of the compressed file directly affects the import speed. It is recommended that the size of the source file after decompressing does not exceed 256 MiB.
```

Comment:
Not so sure
```suggestion
> - Because TiDB Lightning cannot concurrently decompress a single large compressed file, the size of the compressed file directly will affect the import speed. It is recommended that the size of the source file after decompressing does not exceed 256 MiB.
```